README.md

```diff
@@ -65,6 +65,10 @@ To configure the application before installation and usage (see the next section
     `NUM_ASYNC_WORKERS` | Number of workers for course published date async call, defaults to 8
     `CANVAS_URL` | canvas instance url
     `CANVAS_TOKEN` | canvas token for auth
+    `ZOOM_TOKEN` | Token for Zoom
+    `ZOOM_BASE_URL` | Base URL for Zoom
+    `ZOOM_EARLIEST_FROM` | The earliest date to retrieve zoom content from
+    `DEFAULT_SLEEP_TIME` | Amount of time to sleep between retries when given a 429 error and no Retry-After in the response headers.
 
```

Comment:
It looks like 2 types of processes are running for zoom one get the all zoom meeting from canvas courses and another directly from zoom. To me having 2 sentences on the reason for a separate process would be useful. And any Zoom API documentation link will help the next person who comes into this code

Reply:
It's not useful, they should be combined into one thing that takes care of everything for Zoom. This will be related to fixing #33 and #37